spec/libraries/aws_engine_spec.rb

```diff
@@ -75,6 +75,25 @@ def initialize
       end
     end
   end
+
+  module EC2
+    class DummyClient
+      class DummyCredential
+        def access_key_id
+          'iam_access_key'
+        end
+        def secret_access_key
+          'iam_secret_access_key'
+        end
+        def session_token
+          'iam_session_token'
+        end
+      end
+      def config
+        {:credentials => DummyCredential.new}
```

Comment:
__[RuboCop]__

Style/HashSyntax: Use the new Ruby 1.9 hash syntax.


<!-- Comment by [SideCI](https://sideci.com) -->